app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+
+  def comment_update_counter
+    post = Post.find(post_id)
+    post.update_attribute(:comments_count, post.comments_count + 1)
+  end
```

Comment:
[OPTIONAL] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new post has been saved to database, then make the `update_comments_counter` method private

```
  # app/models/comment.rb
  after_save :update_comments_counter

  private

  def update_comments_counter
    # logic here
  end
```